libbeat/docs/breaking.asciidoc

```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
 
 include::./field-name-changes.asciidoc[]
 
+[[breaking-changes-6.7]]
+=== Breaking changes in 6.7
+
+The data structure used to store Beats central management data in Kibana has changed. As a result of the
+template changes, enrolled Beats and configuration tags created prior to 6.7 will not work until you migrate them to the new format.
+UI until their are moved to the new format, we have created a one time [tool](https://github.com/elastic/migrate-management-beats/releases) to help you migrate your data from
```

Comment:
```suggestion
```suggestion
To migrate to the new format, use the https://github.com/elastic/migrate-management-beats/releases[migration tool] we've created.
```
```